src/status_im2/contexts/communities/menus/leave/view.cljs

```diff
@@ -0,0 +1,44 @@
+(ns status-im2.contexts.communities.menus.leave.view
+  (:require [i18n.i18n :as i18n]
+            [quo2.core :as quo]
+            [status-im2.contexts.communities.menus.leave.style :as style]
+            [react-native.core :as rn]
+            [utils.re-frame :as rf]))
+
+(defn hide-sheet-and-dispatch
+  [event]
+  (rf/dispatch [:bottom-sheet/hide])
+  (rf/dispatch event))
+
+
+(defn leave-sheet
+  [id]
+  [rn/view {:style style/container}
+   [rn/view {:style style/inner-container}
+    [quo/text
+     {:accessibility-label :communities-join-community
+      :weight              :semi-bold
+      :size                :heading-1}
+     (i18n/label :t/leave-community?)]]
+   ;; TODO get tag image from community data
+   #_[quo/context-tag
```

Comment:
Zombie code lying around? Why not a GH issue and delete the TODO and zombie code?

Reply:
sure, I'll create a task on this

Reply:
tbh, I prefer to leave the code as the component is correct so it holds the structure which is important.